ErrorStates[] err
float32[] rmse 
string folder

